layers/meta-balena-genericx86/recipes-kernel/linux/linux-yocto_%.bbappend

```diff
@@ -321,10 +324,10 @@ BALENA_CONFIGS_DEPS[tpm] = " \
     CONFIG_SECURITYFS=y \
 "
 BALENA_CONFIGS[tpm] = " \
-    CONFIG_TCG_TPM=m \
-    CONFIG_TCG_TIS_CORE=m \
-    CONFIG_TCG_TIS=m \
-    CONFIG_TCG_CRB=m \
+    CONFIG_TCG_TPM=y \
+    CONFIG_TCG_TIS_CORE=y \
+    CONFIG_TCG_TIS=y \
+    CONFIG_TCG_CRB=y \
```

Comment:
TPMs are not x86 exclusive. It's a standard, and other architectures can use chips that comply with the standard, like https://www.infineon.com/dgdl/Infineon-ISPN-Use-Case-Secured-boot-for-ARM-processor-platforms-ABR-v01_00-EN.pdf?fileId=5546d4625bd71aa0015c0b1fceee0851.
We should move this to meta-balena and make it depend on `SIGN_API`